docs/consensus.md

```diff
@@ -53,6 +53,22 @@ To list all available consensus protocols (with versions), a user can run the co
 
 When a node starts, it sequentially mutates the list of preferred consensus protocols by processing the cluster configuration file and then the mentioned CLI flag. The final list of preferred protocols is then passed to the Priority protocol for cluster-wide consensus. Until the Priority protocol reaches consensus, the cluster will use the default QBFT v2.0 protocol for any duties.
 
+## Consensus Round Duration
+
+There are three diferent round timer implementations. These timers define the duration of each consensus round and how the timing adjusts for subsequent rounds.
+
+### Increasing Round Timer
+
+The `IncreasingRoundTimer` uses a linear increment strategy for round durations. It starts with a base duration and increases by a fixed increment for each subsequend round. The formula for a given round `n` is `Duration = IncRoundStart + (IncRoundIncrease * n)`.
+
+### Eager Double Linear Round Timer
+
+The `EagerDoubleLinearRoundTimer` aligns start times across participants by starting at an absolute time and doubling the round duration when the leader is active. The round duration increases linearly according to `LinearRoundInc`. This aims to fix an issue with the original solution where the leader resets the timer at the start of the round while others reset when they receive the justified pre-prepare which leads to leaders getting out of sync with the rest.
+
+### Exponential Round Timer
```

Comment:
Could you please add CLI flag example of how to enable the specific timer?
Also, mention which one is default and that all nodes of the cluster must use the same consistent setting, otherwise the consensus would fail.

Reply:
Users would likely question themselves what timer to use, would be good to explain a bit what we recommend (the default one) and what difference makes the Exponential timer. Soon when we release this, NOPs will 100% ask this question and we can simply point them to this documentation.